src/components/greetings.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getGreeting } from '../redux/greetings/greeting';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.greetingReducer);
+  let message = '';
+
+  if (status != null) {
+    message = status;
+  } else {
+    message = 'helloo';
+  }
```

Comment:
**OPTIONAL** - A better way to do this will be the `useState` react hook to set your initial message and set the message after your dipstach your action to fetch the greeting from the API. it'd make your code more concise and easier to read. Please fix this 🙏